.circleci/config.yml

```diff
@@ -1,74 +1,298 @@
 version: 2.1
 
+
+executors:
+  py36:
+    docker:
+      - image: shivamshriwas/torchserve:py36
+    environment:
+      _JAVA_OPTIONS: "-Xmx2048m"
+  conda38:
```

Comment:
Make the env names consistent with https://github.com/pytorch/pytorch/blob/master/.circleci/config.yml

Reply:
executor names like torchserve
example: ubuntu18-pythn36-cpu-docker